@gossion In .net sdk build log

```
[MSBuild]      3>Tests/Tests/ContainerServiceOpsTests.cs(28,76): error CS1061: 'ContainerServiceClient' does not contain a definition for 'ContainerServices' and no accessible extension method 'ContainerServices' accepting a first argument of type 'ContainerServiceClient' could be found (are you missing a using directive or an assembly reference?) [/z/work/azure-sdk-for-net/sdk/containerservice/Microsoft.Azure.Management.ContainerService/tests/Microsoft.Azure.Management.ContainerService.Tests.csproj]
```

It's related to test build failure which I assume should be fine?

--- a/custom-words.txt
+++ b/custom-words.txt
@@ -157,6 +157,7 @@ AZFW
 AZNB
 azns
 azpkg
+azpodpolicy
 azureaccounts
 azureactivedirectory
 azureblob

--- a/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/PrivateLinkResourcesList.json
+++ b/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/PrivateLinkResourcesList.json
@@ -1,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "management",
+            "type": "Microsoft.ContainerService/managedClusters/privateLinkResources",
+            "groupId": "management",
+            "requiredMembers": [
+              "management"
+            ],
+            "privateLinkServiceID": "/subscriptions/subid2/resourceGroups/rg2/providers/Microsoft.Network/privateLinkServices/plsName"
+          }
+        ]
+      }
+    }
+  }
+}

--- a/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/ResolvePrivateLinkServiceId.json
+++ b/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/ResolvePrivateLinkServiceId.json
@@ -1,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "parameters": {
+      "name": "management"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "privateLinkServiceID": "/subscriptions/subid2/resourceGroups/rg2/providers/Microsoft.Network/privateLinkServices/plsName"
+      }
+    }
+  }
+}

--- a/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json
+++ b/specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/managedClusters.json
@@ -1280,6 +1280,101 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/privateLinkResources": {
+      "get": {
+        "tags": [
+          "privateLinkResources"
+        ],
+        "operationId": "PrivateLinkResources_List",
+        "summary": "Gets a list of private link resources in the specified managed cluster.",
+        "description": "Gets a list of private link resources in the specified managed cluster. The operation returns properties of each private link resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkResourcesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Private Link Resources by Managed Cluster": {
+            "$ref": "./examples/PrivateLinkResourcesList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/resolvePrivateLinkServiceId": {
+      "post": {
+        "tags": [
+          "resolvePrivateLinkServiceId"
+        ],
+        "operationId": "ResolvePrivateLinkServiceId_POST",
+        "summary": "Gets the private link service ID for the specified managed cluster.",
+        "description": "Gets the private link service ID the specified managed cluster.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkResource"
+            },
+            "description": "Parameters (name, groupId) supplied in order to resolve a private link service ID."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Resolve the Private Link Service ID for Managed Cluster": {
+            "$ref": "./examples/ResolvePrivateLinkServiceId.json"
+          }
+        }
+      }
     }
   },
   "definitions": {
@@ -2983,6 +3078,56 @@
         "description": {
           "type": "string",
           "description": "The private link service connection description."
+        }
+      }
+    },
+    "PrivateLinkResourcesListResult": {
+      "type": "object",
+      "description": "A list of private link resources",
+      "properties": {
+        "value": {
+          "description": "The collection value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        }
+      }
+    },
+    "PrivateLinkResource": {
+      "description": "A private link resource",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the private link resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the private link resource.",
+          "externalDocs": {
+            "url": "https://aka.ms/search-naming-rules"
+          }
+        },
+        "type": {
+          "type": "string",
+          "description": "The resource type."
+        },
+        "groupId": {
+          "type": "string",
+          "description": "The group ID of the resource."
+        },
+        "requiredMembers": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "RequiredMembers of the resource"
+        },
+        "privateLinkServiceID": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The private link service ID of the resource, this field is exposed only to NRP internally."
         }
       }
     }

